dev-support/Jenkinsfile_GitHub

```diff
@@ -144,8 +143,9 @@ pipeline {
                         YETUS_ARGS+=("--docker")
                         YETUS_ARGS+=("--dockerfile=${DOCKERFILE}")
                         YETUS_ARGS+=("--mvn-custom-repos")
-                        YETUS_ARGS+=("--multijdkdirs=/usr/lib/jvm/java-8-openjdk-amd64")
-                        YETUS_ARGS+=("--findbugs-home=/usr")
+                        YETUS_ARGS+=("--java-home=/usr/lib/jvm/java-8")
+                        YETUS_ARGS+=("--multijdkdirs=/usr/lib/jvm/java-8")
```

Comment:
Why do we even specify `--multijdkdirs` ? The different JDK handling is managed at the level of the `Jenkinsfile` pipeline, not within Yetus.

Reply:
Unless the JDK7 builds are silently depending on this? It does look like they install jdk7 and explicitly set `JAVA_HOME` pointed there.

Reply:
But, for instance, `branch-1` has it's own `Dockerfile` and it's own `Jenkinsfile_GitHub` where it *does* make use of `multijdkdirs`. So that's vestigial.

Reply:
I dunno about the PR checker, but for the jira handling job this analysis was correct. The branches-1 build relied on the fact that the docker image set Java home to jdk7.

Multijdk support deduplicates the list so having jdk8 for both home and here should result in a single detected jvm

Reply:
Okay, this was a bit of learning out loud.

My longer-term goal is to converge the Jenkinsfiles/Dockerfiles across branches. Thus the image will ship with multiple JDKs and the jobs will need to manage `JAVA_HOME` the Jenkinsfile instead.